committeeoversightapp/views.py

```diff
@@ -61,8 +61,11 @@ def post(self, request, **kwargs):
             new_category.save()
 
             # if form includes a transcript URL create EventDocument with original and archived url
-            transcript_url = transcript_form.cleaned_data['url']
-            save_document(transcript_url, "transcript", event)
+            documents = [('transcript_url', "transcript"), ('opening_statement_chair', "chair opening statement"), ('opening_statement_rm', "ranking member opening statement")]
+
+            for (field, note) in documents:
+                url = transcript_form.cleaned_data[field]
+                save_document(url, note, event)
```

Comment:
I think your `post` function in the EventCreate view is starting to get a little hefty. I'd like to see some of it broken into modular functions. How about you plan for that in another PR?

Reply:
Sounds good!